Fixed dependency error when running trezorctl on Python 3.6 without rlp.
